content/page/bosc-2025.md

```diff
@@ -2,38 +2,39 @@
 author: nlharris
 cover:
   alt: BOSC-org-committee-Montreal - 1
-  image: /wp-content/uploads/2024/08/BOSC-org-committee-Montreal-1.jpeg
+  image: /obf-hugo-test/wp-content/uploads/2024/08/BOSC-org-committee-Montreal-1.jpeg
```

Comment:
That doesn't look right - if that file is missing we should be uploading it to https://github.com/OBF/wp-content/tree/gh-pages/uploads/2024/08 ?

Reply:
oops, you are right. i copy-pasted this and didn't check that.

Reply:
Logged as #6, we are missing some images from the archive.org rescue - but we have them now 🚀

Reply:
so can this be merged now?

Reply:
Probably OK as a squash-and-merge to keep the history simple, but need to decide if this is a defacto backup live site with edits copied over by hand, or a proof of concept to get the styling done and then re-export the pages from WP and re-import into Hugo?

Reply:
this PR is now out of sync anyway, i will delete. and i don't plan to do other wholesale copy-and-pastes like this!

Reply:
i will delete this PR when i'm done referring to it, ok?

Reply:
Sure.